chain/src/chain.rs

```diff
@@ -505,6 +506,10 @@ impl BlockChain {
 
         storage.save_block_info(block_info.clone())?;
 
+        for (hash_value, write_set) in txn_write_set.iter() {
```

Comment:
we can consume the `txn_write_set`, becase it will be not used any more.

Reply:
Passed it into batch save function